src/util/external/uuid.js

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable no-bitwise */
+// props go to https://gist.github.com/jed/982883
+// NO COPYRIGHTS INVOLVED
```

Comment:
``DO WHAT THE FUCK YOU WANT TO PUBLIC LICENSE
Version 2, December 2004
``

Reply:
Yeeeep http://www.wtfpl.net/txt/copying/